apps/feedahelp/src/pages/api/auth/[...nextauth].ts

```diff
@@ -53,12 +52,12 @@ export const authOptions: NextAuthOptions = {
         const user = {
           ...credentials,
         };
```

Comment:
Nice to know: 
Complexity of Spread Operator is O(n). That means it will iterate over all elements of the array. We could have object assignment which is of O(1). 

Ref: https://stackoverflow.com/questions/57032373/whats-the-time-complexity-of-javascript-spread-syntax-in-arrays

Reply:
check it